---a/net/minecraft/world/chunk/storage/IChunkLoader.java
+++b/net/minecraft/world/chunk/storage/IChunkLoader.java
@@ -1,10 +1,12 @@
 package net.minecraft.world.chunk.storage;
 
 import java.io.IOException;
+
 import net.minecraft.world.MinecraftException;
 import net.minecraft.world.World;
 import net.minecraft.world.chunk.Chunk;
 
-public interface IChunkLoader
-{
+public interface IChunkLoader {
+
     Chunk loadChunk(World worldIn, int x, int z) throws IOException;
 
     void saveChunk(World worldIn, Chunk chunkIn) throws MinecraftException, IOException;
@@ -16,4 +16,4 @@
     void chunkTick();
 
     void saveExtraData();
-}
+}
